modules/eco-poison-pill-about-watchdog.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes/eco-poison-pill-operator.adoc
+
+:_content-type: CONCEPT
+[id="about-watchdog-devices_{context}"]
+= About watchdog devices
+
+Watchdog devices are of two kinds:
+* Hardware watchdog 
+* Software watchdog (or softdog)
```

Comment:
![image](https://user-images.githubusercontent.com/58551443/153139865-3a977575-b681-4827-951d-2b0059956c03.png)
I can see that when switching to a doc view the bullets are not beneath each other